tests/benchdnn/softmax/softmax.cpp

```diff
@@ -232,7 +232,7 @@ void setup_cmp(compare::compare_t &cmp, const prb_t *prb, data_kind_t kind,
     const float trh_coeff_bwd = (prb->dir & FLAG_FWD) ? 1.f : 4.f;
     const float trh_f32 = trh_coeff_log * trh_coeff_bwd * trh_coeff_f32
             * epsilon_dt(trh_dt);
-#if DNNL_AARCH64_USE_ACL || defined(DNNL_SYCL_HIP)
+#if DNNL_AARCH64 || defined(DNNL_SYCL_HIP)
     // MIOpen and ACL softmax accumulate in F16, but oneDNN now expects accumulation in
```

Comment:
This comment needs updating to mention bf16 too

Reply:
Comment updated.